src/borg/helpers/__init__.py

```diff
@@ -0,0 +1,43 @@
+"""
+This package contains all sorts of small helper / utility functionality,
+that did not fit better elsewhere.
+
+Code used to be in borg/helpers.py but was split into the modules in this
+package, which are imported into here for compatibility.
+"""
+
+from .checks import *  # NOQA
+from .convert import *  # NOQA
+from .datastruct import *  # NOQA
+from .errors import *  # NOQA
+from .fs import *  # NOQA
+from .manifest import *  # NOQA
+from .misc import *  # NOQA
+from .msgpack import *  # NOQA
+from .parseformat import *  # NOQA
+from .process import *  # NOQA
+from .progress import *  # NOQA
+from .time import *  # NOQA
+from .usergroup import *  # NOQA
+from .yes import *  # NOQA
```

Comment:
(Overlooked this in my inbox) I'm mistaken here, __all__ isn't useful since there is no `from helpers import *` anywhere. __all__ might have some minor use in the individual helpers.XY modules, to avoid surprises with shadowed names, but it's not anything important, probably not worth it.